api/app/src/routes/webhook/withings.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+
+import { asyncHandler } from "../util";
+import { processData } from "../../command/webhook/withings";
+
+const routes = Router();
+/** ---------------------------------------------------------------------------
+ * POST /webhook/withings
+ *
+ * Receive for all data types for the specified user ID
+ *
+ */
+routes.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    processData(req.body);
+    return res.sendStatus(200);
+  })
+);
+
+/** ---------------------------------------------------------------------------
+ * HEAD /webhook/withings
+ *
+ * Confirm that the webhook was successfully connected
+ *
+ */
+routes.head(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    return res.sendStatus(200);
```

Comment:
Is this part of the Withings high level protocol? Don't we need to respond based on the result of `subscribeToWebhooks()`?

Reply:
This is what the docs say https://developer.withings.com/developer-guide/v3/data-api/keep-user-data-up-to-date/#overview
`When Withings servers send your server a notification, your server needs to respond with a 200 HTTP OK status within two seconds, otherwise the notification delivery will be considered as failed.
Make sure your server responds both to HEAD and POST requests.`